content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Resize Palette Virtual Clusters"
+metaTitle: "Resize Palette Virtual Clusters"
+metaDescription: "Learn how to resize Palette Virtual Clusters"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+
+# Overview
+
+You can resize virtual clusters from the default size of 4 CPU, 4 GiB Memory, 2 GiB Storage. The size you specify can be greater than the Cluster Group allocation. 
+
+# Prerequisite
+
+* A running [Palette Virtual Cluster](/devx/palette-virtual-clusters/pause-restore-virtual-clusters).
+
+# Resize Virtual Clusters
+
+Use the following steps to resize a virtual cluster.
+
+1. Log in to [Palette](https://console.spectrocloud.com).
```

Comment:
Rita, for these bullets, let's add another newline in between them so it looks less compressed when displayed.

Reply:
They each had a new line between them, but I added a break to space them out.